_data/faq.yaml

```diff
@@ -1,20 +1,20 @@
 - question: What is a hackathon?
   answer: A hackathon is an event where motivated people come together and use technology and creativity to build something amazing.
 - question: Who can attend?
-  answer: Any current student! The hackathon is open to designers, writers, computer scientists or anyone interested in hackathons.
+  answer: Any current student! The hackathon is open to designers, writers, computer scientists, or anyone interested in hackathons.
 - question: What should I bring?
   answer: Laptop, chargers, comfortable clothes, and a water bottle. We'll provide you everything else you need.
 - question: Do you offer travel reimbursement?
-  answer: No, everyone attending Local Hack Day should be local. If you're out of state, check out [HackCU V](https://hackcu.org) where we offer reimbursement!
+  answer: No, everyone attending Local Hack Day should be local. If you're out of state, check out [HackCU V](https://hackcu.org), where we offer reimbursement!
```

Comment:
answer: No, everyone attending Local Hack Day should be local. If you're out of state, check out [HackCU VI](https://hackcu.org), where we offer reimbursements!